diff --git a/build/pdf.mjs b/build/pdf.mjs
index f3f03fa38953ebe0b8666c706c751c7e12628752..9dcca9f372547f63b35c9afcaeaa5171932b636c 100644
--- a/build/pdf.mjs
+++ b/build/pdf.mjs
@@ -5766,8 +5766,9 @@ if (isNodeJS) {
     const fs = await import(/*webpackIgnore: true*/"fs"),
       http = await import(/*webpackIgnore: true*/"http"),
       https = await import(/*webpackIgnore: true*/"https"),
-      url = await import(/*webpackIgnore: true*/"url");
-    let canvas, path2d;
+      url = await import(/*webpackIgnore: true*/"url"),
+      canvas = await import( /*webpackIgnore: true*/"canvas");
+    let path2d;
     return new Map(Object.entries({
       fs,
       http,
